component-library/component-lib/src/lib/form-components/checkbox/checkbox.component.html

```diff
@@ -69,16 +48,17 @@
     </div>
     <div aria-live="polite">
       <div
-        *ngIf="formControl?.touched && formControl?.invalid"
+        *ngIf="getErrorState()"
```

Comment:
Can't do this - this will run every tick. You can use a getter if you must use a function